### PR TITLE
Merges statuses and error pages.

### LINKED ISSUE #246: Merge error report and status reports
This would require adding a few additional fields to the status reports.

### PR DESCRIPTION
**Pull request by justinlittman**
closes #246

## Why was this change made?
The reports were redundant.

## Was the documentation (README, DevOpsDocs, wiki) updated?
No.

### CODE REVIEW

File: app/models/status.rb [hunk 1]
```diff
@@ -1,9 +1,13 @@
 # frozen_string_literal: true
 
-Status = Struct.new(:id, :label, :status, :last_update, :update_label, keyword_init: true) do
+Status = Struct.new(:id, :label, :status, keyword_init: true) do
```
Comment (jcoyne) [inline app/models/status.rb]: I was depending on last_update to provide the order for the selected status. How will that work now?

Comment (jcoyne) [reply app/models/status.rb]: It looks like this commit is not in this branch. https://github.com/sul-dlss/google-books/commit/0cb758167801ab84f434388ccdf9836d12c44cf4  Please rebase.

Comment (justinlittman) [reply app/models/status.rb]: How about just sorting on `.updated_at`? Would be simpler and consistent across all statuses.

Comment (jcoyne) [reply app/models/status.rb]: Cool. Rebase and make that change.

Comment (justinlittman) [reply app/models/status.rb]: Done.

File: app/models/book.rb [hunk 2]
```diff
@@ -228,7 +231,7 @@ def self.actual_deposit_rate
   # @param [Integer] page
   def self.with_status(status, page)
     Book.where(status: status.status)
-        .order(status.last_update || 'barcode')
+        .order('updated_at' || 'barcode')
```
Comment (jcoyne) [inline app/models/book.rb]: I don't think the `|| 'barcode'` will be necessary anymore.

Comment (justinlittman) [reply app/models/book.rb]: Removed. We want this descending, right?

**Pull request merged**